pkgs/by-name/lo/longhornctl/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+,
+}:
+buildGoModule rec {
+  pname = "longhornctl";
+  version = "1.7.2";
+
+  src = fetchFromGitHub {
+    owner = "longhorn";
+    repo = "cli";
+    rev = "v${version}";
```

Prefixing tags with `refs/tags` is recommended as sometimes it might clash with branch names
```suggestion
    rev = "refs/tags/v${version}";
```